### ISSUE TITLE
Check skipping MVS

### ISSUE THREAD

**Issue by ErlerPhilipp** [ErlerPhilipp/Pix2Model#160]
Possible speed-up with minor quality impact -> low priority

Check if this is even necessary. MVS in AliceVision (Prepare Dense Scene, Depth Map, Depth Map Filter) doesn't take very long with ~1min.

Meshing should be faster too, which could make it more relevant. AliceVision's Meshing will be impacted a lot by using only the sparse point cloud. So we should do this only after switching to our own meshing.

**Comment by ErlerPhilipp** [comment]
https://colmap.github.io/format.html#text-format
